kormir/src/lib.rs

```diff
@@ -57,8 +58,7 @@ impl<S: Storage> Oracle<S> {
     pub fn from_signing_key(storage: S, signing_key: SecretKey) -> Result<Self, Error> {
         let secp = Secp256k1::new();
 
-        let xpriv_bytes = sha256::Hash::hash(&signing_key.secret_bytes());
-        let nonce_xpriv = ExtendedPrivKey::new_master(Network::Bitcoin, &xpriv_bytes)
+        let nonce_xpriv = ExtendedPrivKey::new_master(Network::Bitcoin, &signing_key.secret_bytes())
```

Comment:
this is a breaking change, before it was a hash of the bytes, now its just the bytes directly